Makefile

```diff
@@ -18,16 +18,16 @@ ifneq ($(findstring ERROR, $(currenttag)),)
     currenttag = 0.0.0
     ifneq ($(stage),)
         currenttag = 0.0.0-$(stage)
-    endif        
-endif        
+    endif
+endif
 
 newtag = $(shell semvertag bump patch $(stagearg))
 ifneq ($(findstring ERROR, $(newtag)),)
     newtag = 0.0.1
     ifneq ($(stage),)
         newtag = 0.0.1-$(stage)
-    endif        
-endif        
+    endif
+endif
 
 registryUrl = registry.service.opg.digital
 oldRegistryUrl = registry.service.dsd.io
```

Comment:
That's the top level Makefile. Are you sure we aren't  using dsd.io anywhere now?